docs/docs-content/user-management/saml-sso/palette-sso-with-entra-id.md

```diff
@@ -0,0 +1,437 @@
+---
+sidebar_label: 'Palette SSO with Microsoft Entra ID'
+title: 'Enable SSO with Microsoft Entra ID'
+description: 'Learn how to set up Palette SSO with Microsoft Entra ID'
+hide_table_of_contents: false
+sidebar_position: 120
+hiddenFromNav: false
+tags: ["user-management", "oidc-sso", "entra-id", "azure-ad"]
+---
+
+
+Single Sign-On (SSO) is an authentication method that enables users to log in to multiple applications and websites with one set of credentials. SSO uses Secure Socket Layer (SSL) certificates to establish and maintain trust between the Service Provider (SP) and an Identity Provider (IDP).
+
+OpenID Connect (OIDC) lets developers authenticate users across websites and apps without owning and managing password files. This technology allows Palette to verify the identity of the person currently using the browser or application.
+
+In October 2023, Microsoft renamed [Azure Active Directory](https://learn.microsoft.com/en-us/entra/fundamentals/new-name) to Microsoft Entra ID. For the purposes of this guide, please consider Microsoft Entra ID and Azure Active Directory to be synonymous. Microsoft Entra ID securely manages anything to do with the user's information, access, and the trust relationships between parties in a flow. It authenticates the user's identity, grants, and revokes access to resources, and issues tokens.
+
+This document will guide you on how to deploy and enabling SSO with Microsoft Entra based on OpenID Connect for integration with Palette and Kubernetes clusters. 
+
+:::tip
+
+If you want to enable OIDC at the Kubernetes cluster level, refer to the [Enable OIDC in Kubernetes Clusters](#enable-oidc-in-kubernetes-clusters) section.
+
+:::
+
+## Architecture
+
```

Comment:
@Jesse-Ben, can you add a high-level summary of what the image below depicts? It's odd to have an image without some accompanying text, especially when numbered.